libraries/apollo-ast/src/test/kotlin/com/apollographql/apollo3/graphql/ast/test/introspection/duplicate.json

```diff
@@ -1512,7 +1512,8 @@
               },
               "defaultValue": null
             }
-          ]
+          ],
+          "isRepeatable": false
```

Comment:
This is (yet another) question: I think we'll want to keep being able to read "old" json schemas. Without this, we're taking the risk of having codegen fail for a lot of users that are using current (or older) spec without `"isRepeatable"`

Reply:
Can you revert that part? I'd like to keep our test suite unchanged unless we make some behaviour change that we need to document either in the CHANGELOG or migration guide (but I don't think this is the case here?)

Reply:
👍 makes sense